### ISSUE TITLE
ERROR Lost connection with Skype... Exiting

### ISSUE THREAD

**Issue by chrise86** [netpro2k/hubot-skype#26]
Get this as soon as I run `bin/hubot -a skype`:

`ERROR Lost connection with Skype... Exiting`

On OSX Mountain Lion, Hubot 2.6.3

**Comment by RafaelGorski** [comment]
Have a look to the following repo:
https://github.com/RafaelGorski/VagrantChefHubot

I manage to configure all of it inside a VM.

**Comment by chrise86** [comment]
Thanks but I'm still having problems. How are you supposed to log Hubot into skype?

> When you first launch hubot with the Skype adapter, Skype will prompt you to allow for API permission. You must allow this before the bot will work.

This doesn't happen...

**Comment by netpro2k** [comment]
@RafaelGorski Wow that vagrant image is awesome.

@chrise86 You need to launch skype and sign into whatever account you would like to be using as a bot. After it is signed in you can launch hubot and you should get the api prompt.

**Comment by fantactuka** [comment]
Probably having similar problem. On OSX, Hubot 2.6.3, with launched Skype when trying to run 

```
export VERSIONER_PYTHON_PREFER_32_BIT=yes # or without it
bin/hubot -a skype  
```

always get crash-report popup "Python quit unexpectedly." and "ERROR Lost connection with Skype... Exiting" in terminal logs.

**Comment by chrise86** [comment]
@fantactuka yes sounds like the same problem, on Mac (not using that vagrant images) I also get that Python error...

**Comment by RafaelGorski** [comment]
@netpro2k ,thanks!
@chrise86 , @fantactuka I just update the readme with revised version of the steps, let me know if works after installing x11(XQuartz), I tested it with MacOSX and Windows 7. 
//Cheers

**Comment by fantactuka** [comment]
@RafaelGorski The issue is **not** related to Vagrant. See @chrise86 comment above

**Comment by RafaelGorski** [comment]
@fantactuka I agree it is **definitely not** related to vagrant, for sure!!! the VM configuration is a practical guide to avoid misconfigurations, well Have a good luck  :)

**Comment by mandar-pathak** [comment]
I am getting the same error. I am wondering if this is an issue with Skype rather than the adapter itself because when starting Skype from the console I get this error:

```
Skype[8211:a07] -[SkypeAPIController init] Failed to set up connection in API controller
```

hubot 2.6.4, OSX Mountain Lion.

**Comment by atipugin** [comment]
Same error here :(

**Comment by RafaelGorski** [comment]
@mandar245  and @atipugin could you please tell me your steps for me? then I can determine the possible fix for the issue. Thanks!

**Comment by mandar-pathak** [comment]
@RafaelGorski I tried to boot an instance of Skype by running this:

`sudo /Applications/Skype.app/Contents/MacOS/Skype`

And saw this in the console:

`013-11-15 17:26:05.627 Skype[93611:a07] -[SkypeAPIController init] Failed to set up connection in API controller`

**Comment by RafaelGorski** [comment]
@mandar245 so if you are using the repo VagrantChefHubot please try to not use 'sudo' and execute all procedure with vagrant user. Maybe if you can copy and past all console commands that you used I can have a better guess what happenning. When I use sudo I got the same problem. 

//cheers 
Rafael Gorski
